### DISCUSSION TITLE
Drag and drop squads to reorder them

### DISCUSSION THREAD

**Discussion by ellie-at-pieces** [dailydotdev/daily#1373] [Feature Requests]
It would be nice if we could drag and drop, or somehow reorder our squads in the side bar. That way you can put your favorite squads near the top so you remember to check them out. This becomes more important as the site grows and users join more squads. 

Pic of me trying to drag Fran's squad to the top where it belongs:
![Screenshot 2024-08-28 121046](https://github.com/user-attachments/assets/5dfe6a1b-34d4-4e8f-a988-69c689a78c18)

**Comment by idoshamun** [comment]
Hi @ellie-at-pieces,

We're working on revamping the squad's section of the sidebar as we speak. So stay tuned 🙏
